app/models/email_template.rb

```diff
@@ -42,6 +42,10 @@ def users
       User.mentee.where.not(registration_token: nil)
     when :unregistered_mentors
       User.mentor.where.not(registration_token: nil)
+    when :abandoned_mails_mentors
+      User.mentor.where(is_missing: true).where(missing_since: ((Time.now - 10.days)..Time.now))   
```

Comment:
@DMips: 
Just to recall: 
There were two types of warning mails: one by last seen > 15 days and one reported by user by pressing button
Therefore after which if the user doesnt appear on website again, we will send an abandoned mail. 
So one condition is when missing in greator than 10 days, which I have inserted. Now for last seen > 25 days, what should be done?  i need to add warning_emails_send_at in db in user table. Should i add ? The query will now send mails to (warning_emails_send_at > 10 days "and" last_actitvity_at > 15 days and missing since > 10 days)